Schema/zrc.zaakInformatieObject.schema.json

```diff
@@ -28,7 +28,8 @@
       "zaak": {
         "$ref": "https://vng.opencatalogi.nl/schemas/zrc.zaak.schema.json",
         "type": "string",
-        "required": true
+        "required": true,
+        "inversedBy": "zaakinformatieobjecten"
```

Comment:
Does this work like this?
Don't you need the uuid of an Attribute or something like that?

Reply:
Yes, this works like this, see the following snippet in the objectReferenceSubscriber:
![image](https://user-images.githubusercontent.com/10433082/217765074-4d122d84-d365-4e56-9124-ba25a70abbf0.png)

Reply:
check